networkx/utils/configs.py

```diff
@@ -11,13 +11,14 @@ class Config:
     """The base class for NetworkX configuration.
 
     There are two ways to use this to create configurations. The first is to
-    simply pass the initial configuration as keyword arguments to ``Config``:
+    simply pass the initial configuration as keyword arguments in a ``Config`` instance:
```

Comment:
We don't pass the arguments to the instance. We pass them to the instance's constructor which is ``Config``. So maybe the original wording is OK?